### PR TITLE
ECOM-1869 adding new attribute credit hours.

### PR DESCRIPTION
**Pull request by awais786**
Adding new attribute credit hour.

### CODE REVIEW

**Comment by awais786** [comment]
@zubair-arbi Please review this PR.

**Comment by zubair-arbi** [comment]
lgtm :+1:

**Comment by awais786** [comment]
@clintonb @wedaly Please review this PR.

File: ecommerce/extensions/catalogue/migrations/0009_credit_hours_attr.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    seat = ProductClass.objects.get(name='Seat')
+
+    # Create our Product Attributes
+    ProductAttribute = apps.get_model('catalogue', 'ProductAttribute')
+    ProductAttribute.objects.create(
+        product_class=seat,
+        name='credit_hours',
+        code='credit_hours',
+        type='integer',
+        required=True
```
Comment (clintonb) [inline ecommerce/extensions/catalogue/migrations/0009_credit_hours_attr.py]: False. Not all seats require credit hours.

**Comment by awais786** [comment]
@clintonb Please review this again.

File: ecommerce/templates/edx/credit/checkout.html [hunk 2]
```diff
@@ -12,7 +12,9 @@
             </div>
             <div class="col-sm-9">
                 <h3 class="title">
-                    Purchasing credit for:
+                    {% with hours=credit_seats.0.attr.credit_hours %}
+                        You are purchasing {{ hours }} credit hour{{ hours|pluralize }} for:
```
Comment (clintonb) [inline ecommerce/templates/edx/credit/checkout.html]: I'm assuming we'll mark these for translation later.

**Comment by clintonb** [comment]
:+1:

**Comment by zubair-arbi** [comment]
@awais786 we will merge this PR after the demo most probably on coming Monday. Since we have to not only modify credit checkout template but also the notifications emails content too.

**Comment by awais786** [comment]
@clintonb please review this PR again. Due to merging made few changes in code.

**Comment by awais786** [comment]
@clintonb branch is not getting green here. Some checks validation is still continue.

File: ecommerce/templates/edx/credit/checkout.html [hunk 3]
```diff
@@ -12,7 +12,9 @@
             </div>
             <div class="col-sm-9">
                 <h3 class="title">
-                    Purchasing credit for:
+                    {% blocktrans with credit_seats.0.attr.credit_hours|pluralize as plural and credit_seats.0.attr.credit_hours as hours %}
```
Comment (clintonb) [inline ecommerce/templates/edx/credit/checkout.html]: Use the `{% plural %}` tag: https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#blocktrans-template-tag.

**Comment by clintonb** [comment]
@awais786 Coveralls is down. Merge once the pluralization issue has been addressed.

**Pull request merged**